### PR TITLE
docs(GiniPayBankSDK)

### PR DESCRIPTION
**Pull request by zladzeyka**
* Add deprecation notice and link to the migration guide.

PIA-1982

### CODE REVIEW

**Review by a-szotyori — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,6 +8,17 @@
 [![Devices](https://img.shields.io/badge/devices-iPhone%20%7C%20iPad-blue.svg)]()
 [![Swift version](https://img.shields.io/badge/swift-5.0-orange.svg)]()
 
+**Deprecation Notice**
+ ----------------------
+
+ The Gini Pay Bank SDK for iOS was replaced by the new
+ [Gini Bank SDK](https://github.com/gini/bank-sdk-ios/.
```
Comment (a-szotyori) [inline README.md:15]: Link format is incorrect: missing closing `)`.
```suggestion
 [Gini Bank SDK](https://github.com/gini/bank-sdk-ios/).
```

File: Documentation/sections/Documentation.md [hunk 2]
```diff
@@ -8,6 +8,17 @@
 [![Devices](https://img.shields.io/badge/devices-iPhone%20%7C%20iPad-blue.svg)]()
 [![Swift version](https://img.shields.io/badge/swift-5.0-orange.svg)]()
 
+**Deprecation Notice**
+ ----------------------
+
+ The Gini Pay Bank SDK for iOS was replaced by the new
+ [Gini Bank SDK](https://github.com/gini/bank-sdk-ios/.
```
Comment (a-szotyori) [inline Documentation/sections/Documentation.md:15]: Link format is incorrect: missing closing `)`.
```suggestion
 [Gini Bank SDK](https://github.com/gini/bank-sdk-ios/).
```

**Review by a-szotyori — CHANGES_REQUESTED** [review]

**Comment by zladzeyka** [comment]
I've verified the link on the migration guide.

**Review by a-szotyori — APPROVED** [review]
👍

**Pull request merged**